lib/raven/interfaces/exception.rb

```diff
@@ -4,15 +4,12 @@ module Raven
   class ExceptionInterface < Interface
 
     name 'exception'
-    attr_accessor :type
-    attr_accessor :value
-    attr_accessor :module
-    attr_accessor :stacktrace
+    attr_accessor :values
```

Comment:
Docs should be fixed, but this looks correct to me (values is a list of exception instances)